src/Uno.CrossTargetting.targets

```diff
@@ -136,9 +136,9 @@
 			<!-- Current nuget publishing uses netX.0-android30.0 -->
 			<_OverrideTargetFramework Condition="$(_OverrideTargetFramework.EndsWith('.0-android'))">$(_OverrideTargetFramework)30.0</_OverrideTargetFramework>
 
-			<_overridePackageId>$(CommonOverridePackageId)</_overridePackageId>
+			<_overridePackageId>$(CommonOverridePackageId.ToLowerInvariant())</_overridePackageId>
 			<_overridePackageId Condition="'$(CommonOverridePackageId)'==''">$(AssemblyName)</_overridePackageId>
-			<_overridePackageId Condition="'$(UNO_UWP_BUILD)'=='false'">$(_overridePackageId.Replace('Uno.UI','Uno.WinUI'))</_overridePackageId>
+			<_overridePackageId Condition="'$(UNO_UWP_BUILD)'=='false'">$(_overridePackageId.Replace('uno.ui','uno.winui'))</_overridePackageId>
```

Comment:
This is not going to work when `AssemblyName` is used.
Either do ToLowerInvariant on AssemblyName, or keep `_overridePackageId the same and adjust it in `_TargetNugetFolder` (lines 142 and 143)